### DISCUSSION TITLE
Repeated Export to Excel/Google Sheets file?

### DISCUSSION THREAD

**Discussion by denysullivan** [AutomaApp/automa#61] [Q&A] [answered]
I'm scraping some website data using the interval trigger (every hour). Currently this is setup to export as CSV giving me a new download every hour. Is there a way to compile this data easily? It would be nice to be able to essentially drop it into a google sheets file if that's possible?

**Comment by Kholid060** [comment, marked as answer]
There's no way to do this automatically, so you have to do it manually
